frontend/src/component/project/ProjectAccess/ProjectAccessTable/ProjectAccessTable.tsx

```diff
@@ -490,11 +488,17 @@ export const ProjectAccessTable: VFC = () => {
                 setOpen={setGroupOpen}
                 group={selectedRow?.entity as IGroup}
                 projectId={projectId}
-                subtitle={`Role: ${
-                    access?.roles.find(
-                        ({ id }) => id === selectedRow?.entity.roleId
-                    )?.name
-                }`}
+                subtitle={
+                    <>
+                        {selectedRow && selectedRow.entity.roles.length > 1
+                            ? 'Roles:'
+                            : 'Role:'}
```

Comment:
Will there always be >= 1 roles? Or could selectedRow.entity.roles.length be 0 and should it then say "Roles:" if there are 0 roles in selectedRow.entity.roles ?

Reply:
I think this is safe. This is shown inside project access and only when you click in a group that has access, which means they should have a role. Worst case scenario it would display "Role:      ", which doesn't sound super bad.